src/seedu/addressbook/Main.java

```diff
@@ -84,9 +84,13 @@ private void runCommandLoopUntilExitCommand() {
         do {
             String userCommandText = ui.getUserCommand();
             command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
+            try {
+                CommandResult result = executeCommand(command);
+                recordResult(result);
+                ui.showResultToUser(result);
+            } catch(StorageOperationException soe){
+                ui.showToUser(soe.getMessage());
```

Comment:
In general, just showing the error message or printing the stack trace is not considered "good exception handling".
Think how you can improve the user experience in light of the exception that just occurred.